_templates/event.php

```diff
@@ -9,13 +9,22 @@
 
 $l10n->set_domain('layout');
 
-if (event_active('odin 6.0 release')) { ?>
+const EVENT_NAME = 'odin 6.0 release';
+
+if (event_active(EVENT_NAME)) { ?>
   <div class="overlay">
     <div class="overlay__content toast">
       <div class="toast__close"><i class="fas fa-times"></i></div>
-      <span class="toast__text"><strong>Something new is coming.</strong> Check back Tuesday after 1600 UTC.</span>
+      <span class="toast__text"><strong>Something new is coming.</strong> Check back <time id="odin-day">Tues</time> after <time id="odin-time">1600 UTC</time>.</span>
     </div>
   </div>
+  <script>
+    const utcDate = new Date("<?php echo date(DATE_ISO8601, date_timestamp_get($event_expires[EVENT_NAME][1])) ?>");
```

Comment:
Is there any benefit to winning the Date in PHP as opposed just making it a JavaScript date?

Reply:
I mean we already have the date in PHP so I was just trying to be consistent, especially in case we were to change it (as unlikely as that is).